lib/core/Canvas.js

```diff
@@ -401,11 +401,17 @@ Canvas.prototype.setActivePlane = function(plane) {
     }
   }
 
+  var previousPlane = this._activePlane;
+
   // hide previous Plane
-  if (this._activePlane) {
+  if (previousPlane) {
     svgAttr(this._activePlane.layer, 'display', 'none');
-  }
 
+    // svg should only have one associated element
```

Comment:
Could you elaborate on this? Why are we doing what we're doing here?

Reply:
We have a link from the svg to the currently active root element (`data-element-id="..."`), which we update on plane switch: https://github.com/bpmn-io/diagram-js/blob/91626d03b756573951c687b72ff21b81c0a3db80/lib/core/Canvas.js#L421

We also have a reference from the element to the svg, which we did not update on plane switches previously.

When we removed an (inactive) plane + root element, it would also remove the `data-element-id` from the svg, even when it was not the "active" element. This breaks mouse interaction with the diagram.

Reply:
Which exact _mouse interaction_? Just trying to understand!

Reply:
Specifically scrolling the canvas. The events are captured here:
https://github.com/bpmn-io/diagram-js/blob/1063f7c18474096d3d7c9e400ce82a1bf762a157/lib/features/interaction-events/InteractionEvents.js#L82-L92

for the root, this would lead to 
 * `gfx === canvas._svg`
 * and because of `data-element-id === ""` ==> `element === null`

it would not find an element and not fire mouse events

Reply:
Got it. Makes sense (and is obviously a bug!).